src/jit/codegencommon.cpp

```diff
@@ -9450,6 +9450,13 @@ void CodeGen::genFnEpilog(BasicBlock* block)
     genRestoreCalleeSavedFltRegs(compiler->compLclFrameSize);
 #endif // !FEATURE_STACK_FP_X87
 
+#ifdef JIT32_GCENCODER
+    // On IA32, we start the OS-reported portion of the epilog after restoring any callee-saved
+    // floating-point registers. This avoids the need to update the x86 unwinder and retains binary
+    // compatibility between later versions of the JIT and earlier versions of the runtime.
+    getEmitter()->emitStartEpilog();
```

Comment:
IMO, this comment is confusing. If there _were_ any callee-saved registers, then of course the unwinder would have to know about them. So the real, and only, reason is that we need to avoid including in the VM-known epilog, instructions that the unwinder doesn't understand, and which don't contribute to unwinding, namely vzeroupper. If we were to change the ABI convention to include callee-saved xmm/ymm registers (which perhaps we should do, for CQ/performance reasons), we'd need to update the unwinder, and move this line of code.